packages/react-native-editor/ios/Podfile.lock

```diff
@@ -306,7 +306,7 @@ PODS:
     - React-Core
   - RNGestureHandler (2.2.0-wp-4):
     - React-Core
-  - RNReanimated (2.2.4-wp-1):
+  - RNReanimated (2.4.1-wp-1):
```

Comment:
I wonder about this change, because the `package.json` does have the `2.4.1` version, so maybe the `Podfile.lock` wasn't updated? 🤔

Reply:
The version is correct in `trunk` so most likely we won't see this change when updating the branch before merging it.

https://github.com/WordPress/gutenberg/blob/01ab53707924a02940ca43f4fd1f048bda261335/packages/react-native-editor/ios/Podfile.lock#L309

Reply:
Right, just checked that in the `1.72.0` tag, it has the right version as well so its good. Had to double check 😄